.drone.yml

```diff
@@ -15,24 +16,6 @@ steps:
   - git remote set-branches --add origin $DRONE_TARGET_BRANCH
   - git fetch --depth 100 origin $DRONE_TARGET_BRANCH
 
-- name: verify-pr
```

Comment:
Removing this logic because I saw this Step intermittently failing and because there's now a native feature in Drone server that provides this functionality.

https://drone.cdn-code.org/code-dot-org/code-dot-org/settings
<img width="536" alt="image" src="https://user-images.githubusercontent.com/2157034/93115451-b57f2480-f670-11ea-89ce-967e3ed52d90.png">

Reply:
That's great, I never liked having so much logic in the `.drone.yml` file.